security/2.0/_index.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Jakarta Security 2.0 (milestone)"
-date: 2020-06-11
+title: "Jakarta Security 2.0"
+date: 2020-09-02
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
 
 * [Jakarta Security 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.es/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Security 2.0 Specification Document](./security-spec-2.0.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Security 2.0 Specification Document](./security-spec-2.0.0-SNAPSHOT.html) (HTML)
+* [Jakarta Security 2.0 Specification Document](./jakarta-security-spec-2.0.0-SNAPSHOT.pdf) (PDF)
```

Comment:
The docs that are generated are called `jakarta-security-spec-2.0.0.pdf` and `security-spec-2.0.0.html`, so without the jakarta prefect and without snapshot.

Reply:
I have renamed the files and make sure the links were correct and without snapshot.
Now we need to generate the apidocs and spec docs and update the PR.

Do you want me to do that?

Reply:
Yes, sure ;) It's just a matter of running `mvn clean install -Dstatus=Final` in the project, and setting the version to 2.0.

Reply:
I'll do then